changelogs/fragments/1661-job-owner-valid-characters.yml

```diff
@@ -0,0 +1,9 @@
+bugfixes:
+  - zos_job_output - RACF user names containing a ``@`` raised an invalid
```

Comment:
I think you can add another sample of failing character or just say it didn't work with other special characters.

`RACF user names containing a ``@`` or other special characters raised an invalid argument error.`